.github/workflows/immutable-sqitch-change.yml

```
@@ -0,0 +1,16 @@
+name: Test sqitch changes immutability
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
+
+jobs:
+  check_immutable_files:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: cd ~/cas-ciip-portal; .bin/sqitch-check-immutable-files.sh schema origin/master
```

```suggestion
      - run: .bin/sqitch-check-immutable-files.sh schema origin/master
```
I don't think we need the directory change here